bazel_tools/scala.bzl

```diff
@@ -48,7 +48,6 @@ common_scalacopts = version_specific.get(scala_major_version, []) + [
     # doesn't allow advance features of the language without explict import
     # (higherkinds, implicits)
     "-feature",
-    "-target:jvm-1.8",
```

Comment:
do you need to remove this ?

Reply:
that scalac option is deprecated in 2.13.10. I would trepidatiously suggest to drop it all together unless there is a good reason not to do this.